pkg/resources/management.cattle.io/v3/clusterroletemplatebinding/ClusterRoleTemplateBinding.md

```diff
@@ -0,0 +1,28 @@
+## Validation Checks
+
+### Escalation Prevention
+
+Users can only create/update ClusterRoleTemplateBindings which grant permissions to RoleTemplates which rights less than or equal to those they currently possess. This is to prevents privilege escalation. 
+
+### Invalid Fields - Create
+
+Users cannot create ClusterRoleTemplateBindings which violate the following constraints:
+- Either a user subject (through "UserName" or "UserPrincipalName") or a group subject (through "GroupName" or "GroupPrincipalName"), but not both, must be specified
+- A "ClusterName" must be specified
+- The roleTemplate indicated in "RoleTemplateName" must be:
+  - Valid (i.e. is an existing `roleTemplate` object in the `management.cattle.io/v3` apiGroup)
+  - Not locked (i.e. `roleTemplate.Locked` must be `false`)
+
+### Invalid Fields - Update
+
+Users canot update the following fields after creation:
+- RoleTemplateName
+- ClusterName
+
+Users can update the following fields if they have not been set, but after they have been set they can't be changed:
```

Comment:
This is super nit picky, but for consistency with the rest of the file:
```suggestion
Users can update the following fields if they have not been set, but after they have been set they cannot be changed:
```

Reply:
Change Made.